### DISCUSSION TITLE
Any work going to update this to Schwab

### DISCUSSION THREAD

**Discussion by dougbrowning** [areed1192/td-ameritrade-python-api#258] [General]
I had been using this lib for 3+ years and it was working great until Schwab shut it down.  I am finally now after months getting emails I can get a Schwab developer account and the API is out.  Any plans to update this lib to the new Schwab?  I got a lot of work into the code and would love to get it running again.

thanks a lot

**Comment by cmrncrick** [comment]
I don't think Alex is very active anymore with this. I wish he had plans on updating as I, too, use this extensively.
